app-network/easytier/autobuild/defines

```diff
@@ -0,0 +1,15 @@
+PKGNAME=easytier
+PKGSEC=net
+PKGDES="A decentralized VPN tunnel"
+PKGDEP="gcc-runtime glibc"
+PKGRECOM="systemd"
```

Comment:
This is meaningless - there is no way that you will be able to assemble an AOSC OS sysroot without systemd. I would just call it a dependency.